### ISSUE TITLE
System dependencies not installed?

### ISSUE THREAD

**Issue by scythargon** [dtolstyi/node-chromium#52]
Hello, after I install  node-chromium and try to verify it via 

`node_modules/chromium/lib/chromium/chrome-linux/chrome --version`

I get:

`libgobject-2.0.so.0: cannot open shared object file: No such file or directory`

**Comment by dtolstyi** [comment]
This is the issue with your system, unfortunately. Chromium to be launched needs quite some libraries to be present in system. You can find relevant information in specific articles or StackOverflow.
This package provides just binary to launch chromium but it doesn't install dependencies.
After you prepare your system you should be able to launch chromium as well. You can try launching usual Chrome / Chromium at your system first by installing them and if they work then this package binaries should work as well

**Comment by scythargon** [comment]
Thank you!

Let me share my discoveries here for ones who might stumble upon the same problem.

Yup, I've tried to install chromium from ubuntu repos, but it didn't fix the issue, probably I would have more chances if I installed google-chrome-stable from some PPA, but then there might be a versions issue since PPA does not have older chrome versions (which I install via node-chromium).

Then comparing to a system (basically - docker image) which had Chrome installed fine I've gathered a list of libraries it might need, so the short version which was enough it my case is

`apt install libcairo-gobject2 libatk-bridge2.0-0 libnss3 libcups2 libgbm1 libpango-1.0-0 libasound2 libxkbcommon0` 

But then after thinking why it was working before - I've realized that it's the `playwright` package which was updated recently, which was installing all the required libs for me earlier and now it would not do that.

So eventually I've solved it by replacing 

`npx playwright install-deps`

via 

`npx playwright@1.17.1 install-deps`

which install the libraries I need for my version of chrome  (Chromium 96.0.4664.0 or NODE_CHROMIUM_REVISION=929513)
